js/viz/chart.js

```diff
@@ -467,6 +467,13 @@ const dxChart = AdvancedChart.inherit({
         this._correctValueAxes(true);
     },
 
+    _setDeprecatedOptions() {
+        this.callBase();
+        _extend(this._deprecatedOptions, {
+            'argumentAxis.aggregateByCategory': { since: '23.1', message: 'This option is deprecated, use the aggregation.enabled property instead' }
```

Comment:
1. We shouldn't write "This option is deprecated" in this message. Now full logged message looks like `error.js:24 W0001 - dxChart - 'argumentAxis.aggregateByCategory' option is deprecated in 23.1. This option is deprecated, use the aggregation.enabled property instead. See:
http://js.devexpress.com/error/23_1/W0001`. 
2. This message is not fully truthful. "aggregation.enabled" is not a replacement for "aggregationByCategory". We discussed it with a tech writer.

Let's change it to 'Use the "aggregation.enabled" property instead'